<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WtEditorClass</class>
 <widget class="QMainWindow" name="WtEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wt Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Resource manager</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="5">
         <widget class="QTabWidget" name="managersTabWidget"/>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>706</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="logDock">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QListWidget" name="logList"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../vs10/Wt/WtEditor/WtEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>WtEditorClass</receiver>
   <slot>onBtnSaveClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>WtEditorClass</receiver>
   <slot>onOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>WtEditorClass</receiver>
   <slot>onNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>WtEditorClass</receiver>
   <slot>onReload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnSaveClick()</slot>
  <slot>onTreeContextMenu(QPoint)</slot>
  <slot>onTreeItemActivated(QTreeWidgetItem*,int)</slot>
  <slot>onNew()</slot>
  <slot>onBtnClearAllClick()</slot>
  <slot>onOpen()</slot>
  <slot>onReload()</slot>
 </slots>
</ui>
